bootstrap/kubeadm/internal/bottlerocket/bootstrap.go

```diff
@@ -22,14 +16,26 @@ provider-id = "{{.ProviderId}}"
 {{- end -}}
 `
 
-	bootstrapHostContainerTemplate = `{{define "bootstrapHostContainerSettings" -}}
-[settings.host-containers.kubeadm-bootstrap]
+	hostContainerTemplate = `{{define "hostContainerSettings" -}}
+[settings.host-containers.{{.Name}}]
 enabled = true
-superpowered = true
-source = "{{.BootstrapContainerSource}}"
-user-data = "{{.BootstrapContainerUserData}}"
+superpowered = {{.Superpowered}}
+{{- if (ne (imageUrl .ImageMeta) "")}}
```

Comment:
Why do we have a conditional on the source or userdata? Is it possible for those fields to not be set for a host-container?

Reply:
Yes. The `admin` and `control` container can have an empty source and it will use the default value that will depend on BR's version. However, for custom containers this value must be specified but in order to make the code generic and reusable, I use the same template for both custom containers and the default containers.

User data is optional in BR.